infra/terraform/test-org/org/locals.tf

```
@@ -711,5 +711,14 @@ locals {
       owners      = ["Saipriyavk", "ChrisBarefoot"]
       topics      = local.common_topics.net
     },
+    {
+      name            = "terraform-google-gen-ai-document-summarization"
+      short_name      = "gen-ai-document-summarization"
+      org             = "GoogleCloudPlatform"
+      description     = "Summarizes document using OCR and Vertex Generative AI LLM"
+      owners          = ["asrivast", "balajismaniam", "erschmid", "liyvonne", "nicain"]
```

Please use github usernames

```suggestion
      owners          = ["asrivas", "balajismaniam", "telpirion", "yil532", "nicain"]
```